s4e-web/src/app/state/session/session.query.ts

```diff
@@ -34,6 +34,18 @@ export class SessionQuery extends Query<Session> {
   isAdmin() {
     return this.getValue().admin;
   }
+
+  canDeleteInstitution() {
+    return this.getValue().authorities
+      .some(authority => authority === 'OP_INST_DELETE')
```

Comment:
for arrays of strings you can just use `.authorities.includes('OP_INST_DELETE')`